### ISSUE TITLE
Use Paths for aws cli or aws-iam-authenticator for EKS AuthN - MacOS

### ISSUE THREAD

**Issue by jimmyraywv** [kubeshop/monokle#1285]
**Describe the enhancement you'd like to see**
In my `.kube/config` I use either `aws eks get-token` or `aws-iam-authenticator` as the commands to authenticate to EKS clusters.

```
...
      args:
      - token
      - -i
      - monokle
      command: aws-iam-authenticator
...
```

or

```
...
      command: aws
      args:
        - "eks"
        - "get-token"
        - "--cluster-name"
        - "monokle"
 ...
```

The tools that create the `user` elements in the `.kube/config` do not include the absolute paths for these binaries. My shell resolves the paths. It would be very handy for Monokle to resolve the paths as well. This would reduce troubleshooting, and the need to continually edit `kube/config` files `user` elements.

Without the absolute paths (`which aws` or `which aws-iam-authenticator`) the AuthN fails with the following error:
```
Error: Command failed with ENOENT: aws --region <REGION> eks get-token --cluster-name <CLUSTER_NAME> spawnSync aws ENOENT
```

**Comment by olensmar** [comment]
Thanks for reporting @jimmyraywv - will have investigate the root cause of this...

**Comment by olensmar** [comment]
@jimmyraywv just following up - this should be fixed in the recent 1.5.1 release - thanks for reporting!
